content/docs/command-reference/push.md

```diff
@@ -95,6 +95,8 @@ in the cache (compared to the default remote.) It can be used to see what files
 - `-j <number>`, `--jobs <number>` - parallelism level for DVC to upload data to
   remote storage. The default value is `4 * cpu_count()`. For SSH remotes, the
   default is `4`. Using more jobs may improve the overall transfer speed.
+  Alternatively, a custom value could be set in a configuration file using
+  `jobs` option. See `dvc remote modify` for more information.
```

Comment:
Also, this needs to be added to every command that supports `--jobs`. E.g. `dvc fetch/status/push`.

Reply:
Yep, I see. They are consistent at this point.

Reply:
How about
```suggestion
  The default value can be set with the
  `jobs` config option in `dvc remote modify`.
```
> Needs formatting though

And probably keep the sentence "Using more jobs may improve the overall transfer speed." at the end of the paragraph.